js/src/setup-mc/setup-stepper/setup-paid-ads/setup-paid-ads.js

```diff
@@ -56,6 +58,19 @@ export default function SetupPaidAds() {
 		await finishFreeListingsSetup( event );
 	};
 
+	function createSkipButton( text ) {
+		return (
+			<AppButton
+				isTertiary
+				data-action="skip-ads"
+				text={ text }
+				loading={ completing === 'skip-ads' }
+				disabled={ completing === 'complete-ads' }
```

Comment:
💅  WDYT Of using some  constants here?

Reply:
Considering the following two PRs are also changing related code fragments, I updated it in PR #1655 by https://github.com/woocommerce/google-listings-and-ads/commit/0e65308cf6ecb3163adadd39530172ffdaf8ef81.